addons/hr_expense/static/src/scss/hr_expense.scss

```diff
@@ -34,3 +34,7 @@
       height: 100%;
     }
 }
+
+td[name="description"] p:last-child {
```

Comment:
Does this only apply to the list view/model where we want to?
Or could it apply anywhere in Odoo?

Reply:
It's going to be applied on any view/model inside the hr_expense module which is not right. Will create a class and apply that class only on the list we want. 
About the second question, I'm not sure but right now without creating the class, I think it should only be applied to the hr_expense module, right?